app/src/main/java/edu/grinnell/grinnell_publications_android/Models/Interfaces/Story.java

```diff
@@ -0,0 +1,58 @@
+package edu.grinnell.grinnell_publications_android.Models.Interfaces;
+
+
+import java.util.List;
+
+/**
+ * @author Albert Owusu-Asare
+ * @since 1.1 Thu May  5 23:29:12 CDT 2016
+ * @see Publication
+ * @see Series
+ * @see Author
+ * @see Reactions
+ */
+public interface Story {
+    /**
+     * Returns all the @code{publication}s the story belongs to.
+     *
+     * <p>If the story belongs to a single @code{publication} then the size of the list returned is
+     * 1.</p>
+     *
+     * @return the publication the story belongs to.
+     */
+    List<Publication> getPublication();
```

Comment:
As per our current API spec, each article object contains a field for a publication object, which will have limited info like a thumbnail for a logo, title and id.

We will probably use that id to query our publications endpoint to retrieve additional info for that publication if we wish.